collect_app/src/main/java/org/odk/collect/android/fastexternalitemset/ItemsetDbAdapter.java

```diff
@@ -1,11 +1,12 @@
-package org.odk.collect.android.database;
```

Comment:
I think introducing a new package to group what you have is a great idea. I wonder whether it should be top-level, though. It could be in `formentry` or `widgets`. While here, I wonder what the `itemset` package and `Itemset` class were supposed to be for. Seems they can be removed.

Reply:
I added it there because we have also external package it's top-level too which contains classes responsible for reading external csv files we use to handle search/pulldata.

> While here, I wonder what the itemset package and Itemset class were supposed to be for. Seems they can be removed.

Yes it's unused. I added it implementing storage migration to handle migrating itemsets databases but then we decided not to do that because we can clear them just and such databases will be recreated and probably I forgot to remove it.